packages/tupaia-web/index.html

```diff
@@ -112,12 +112,43 @@
       background: #262834;
       color: white;
     }
+    @keyframes spinner {
```

Comment:
this isn't really related but I took the chance to add an initial loader to the page